packages/imperative/__tests__/src/packages/imperative/plugins/test_plugins/override_plugin/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "override-plugin",
+  "name": "@zowe/secrets-for-kubernetes-for-zowe-cli",
```

Comment:
no objections on this, but not sure we need to modify the name of this test plugin 😋

Reply:
Historically, I think one of the tests changes it, and if something goes wrong, it gets left over as an artifact.